README.md

```diff
@@ -52,6 +52,7 @@ module.exports = function(environment) {
   };
 ```
 
+<<<<<<< HEAD
```

Comment:
Looks like unresolved conflict

Reply:
Oof! Embarrassing. Fixed now.

Reply:
Not embarrassing at all - thanks for taking the time to move this addon forward :)